src/app/services/xml-parsers/generic-parser.service.ts

```diff
@@ -121,6 +122,18 @@ export class GenericParserService {
     };
   }
 
+  private parsePtr(xml: XMLElement) {
+    if (xml.getAttribute('type') === 'noteAnchor' && xml.getAttribute('target')) {
+      const noteId = xml.getAttribute('target').replace('#', '');
```

Comment:
change `replace` with `slice`?

Reply:
I prefer `replace`, because if someone forget to use the "#" before the referenced id, the transformation still works.